packages/autometrics-typescript-plugin/src/astHelpers.ts

```diff
@@ -105,7 +105,7 @@ export function getNodeIdentifier(
 export function getNodeType(node: ts.Node, typechecker: ts.TypeChecker) {
   const declaration = typechecker.getSymbolAtLocation(node);
 
-  if (!(node.parent || declaration.valueDeclaration)) {
+  if (!(declaration.valueDeclaration && node.parent)) {
```

Comment:
Isn't this the same as:
```suggestion
  if (!declaration.valueDeclaration && !node.parent) {
```
Which is a bit shorter and a bit easier to read (imo)

Reply:
it is in my view as well - Rome linter was just complaining about it "unnecessary complexity in logic" :)

Reply:
happy to ignore Rome and go ahead with your suggested option though

Reply:
Interesting. I don't really like disabling linter rules either. But disabling the "unnecessary complexity in logic" for this line here is ok. I will approve the PR anyway. I'm fine either way.